package.json

```diff
@@ -3,14 +3,17 @@
   "version": "5.8.1",
   "description": "react tooltip component",
   "scripts": {
-    "dev": "node ./cli.js --env=development && node --max_old_space_size=2048 ./node_modules/rollup/dist/bin/rollup -c rollup.config.dev.js --watch",
-    "build": "node ./cli.js --env=production && npm run types && node --max_old_space_size=2048 ./node_modules/rollup/dist/bin/rollup -c rollup.config.prod.js && npm run bundlesize",
+    "dev-rollup": "node ./cli.js --env=development && node --max_old_space_size=2048 ./node_modules/rollup/dist/bin/rollup -c rollup.config.dev.js --watch",
+    "build-rollup": "node ./cli.js --env=production && npm run types && node --max_old_space_size=2048 ./node_modules/rollup/dist/bin/rollup -c rollup.config.prod.js && npm run bundlesize",
+    "dev": "node ./cli.js --env=development && node ./esbuild.config.dev.mjs",
+    "build": "node ./cli.js --env=production && node ./esbuild.config.prod.mjs",
```

Comment:
Just keeping two bundlers available because of comparison tests

Reply:
also, using the CLI because esbuild doesn't clear the build/dist folder, only updates the existent files, so, the manual file to keep the folder always being cleared.

PS: I think we should rename this file, a better name than `cli.js`